stream_chat/tests/test_channel.py

```diff
@@ -147,6 +147,11 @@ def test_send_and_delete_image(self, channel, random_user):
         assert "lena.png" in resp["file"]
         # resp = channel.delete_image(resp['file'])
 
+    def test_send_image_with_bot_blocked(self, channel, random_user):
```

Comment:
What exactly are you testing here?

Reply:
Normally that url would be blocked, with changes we can grab the content. Open to suggestions for improvement.

Reply:
So it is common for some sites, twilio among them, to block 'requests' or 'python' as a user-agent and chat users want to be able to link to images wherever. That is the problem being solved right? If so maybe look into sth like this:  https://pypi.org/project/random-user-agent/

Reply:
Yes.

I didn't want to add a dependency. Also, a good detector would understand it anyway. That's why added a generic one to bypass analytics tracking.

Reply:
I agree that adding a dependency sucks and that it's mostly an arms race. Maybe shouldn't get into it all? Catch 400 errors and notify the user?

Reply:
I would prefer going with this since it seems it will fix for main trackers.

Otherwise, failure modes are also tracky (400, 401, 403 etc.) If a url would fail with this, it should fail without this change too.

Reply:
Maybe we should add a note/link to this discussion in the test so we don't get into the arms race if this starts failing in a year's time?

Reply:
Sounds good, adding.